test/api.js

```diff
@@ -19,6 +19,21 @@ describe('client-side api', function () {
 				.call(done);
 		});
 
+		it('should ensure that duplicate bundleName-messageName pairs are ignored', done => {
+			e.apis.extension.listenFor('testMessageName', 'testBundleName', done);
+			e.apis.extension.listenFor('testMessageName', 'testBundleName', done);
```

Comment:
Huh, wow yeah it just logs but doesn't `throw` lol. I've just made a commit that makes this actually `throw` now, can you rebase this branch and adjust the test to use `assert.throws`?